codespell_lib/data/dictionary.txt

```diff
@@ -33534,6 +33534,10 @@ kyeboshed->kiboshed
 kyeboshes->kiboshes
 kyeboshing->kiboshing
 kyrillic->cyrillic
+labael->label
+labaeled->labeled, labelled,
+labaeling->labeling, labelling,
```

Comment:
Perhaps add British spelling?
```suggestion
labaeling->labeling, labelling,
labaelled->labelled, labeled,
labaelling->labelling, labeling,
```

Reply:
Added, sorry but I couldn't get GitHub to commit the suggestion so no credit.